tests/default_test.go

```diff
@@ -0,0 +1,82 @@
+package test
+
+import (
+	"API-weather/libraries"
```

Comment:
The import for the unit test must look something like this:
```
import (
	"encoding/json"
	"fmt"
	"net/http"
	"net/http/httptest"
	"path/filepath"
	"runtime"
	"testing"

	"github.com/kaiijimenez/API/libraries"

	_ "github.com/kaiijimenez/API/routers"

	"github.com/astaxie/beego"
	. "github.com/smartystreets/goconvey/convey"
)
```

To be executed with this command:
```
go test github.com/kaiijimenez/API/... -v
```

Also rename this file to `weather_test.go`. Don't forget to removed `tests/conf/`

Reply:
test/conf/ deleted and calling conf.app from root